src/views/tools/image_management.test.ts

```diff
@@ -13,11 +12,55 @@ import MemoryStoragePortal from "../../server/services/storage_portal/storage_po
 import { timeout } from "../../shared/utils";
 import { getAllImageInfos, startUpdatingImages } from "./image_management";
 import { ImageInfo } from "./types";
+import { spawn } from "child_process";
+import { Level, setLogLevel } from "../../server/log";
+
+// Helper to save a jpeg buffer to a file, and run imagemagick's identify tool on it.
+function identify(jpgData: Buffer): Promise<string> {
+  return new Promise((resolve) => {
+    // TODO: Remove dependency on the filesystem, see if there's a way to send the data
+    // through a stdin stream or something.
+    const tmpFile =
+      "/tmp/imagemanagement_img_" + Math.floor(Math.random() * 100) + ".jpg";
```

Comment:
What's the purpose of calling Math.floow & Math.random? Why not overwrite a singular filename each time?